wire-ios-sync-engine/Support/Sourcery/generated/AutoMockable.manual.swift

```diff
@@ -100,3 +100,129 @@ public class MockSessionEstablisherInterface: SessionEstablisherInterface {
     }
 
 }
+
+public class MockMessageAppendableConversation: MessageAppendableConversation {
```

Comment:
Why did this need to be manually mocked?

Reply:
It was complicated to do it automatically that's why we decided to do it manually with @caldrian. Maybe in the future we can decide to make it automatic.